src/travel-details-map-screen/TravelDetailsMapScreenComponent.tsx

```diff
@@ -161,28 +182,42 @@ const LiveVehicle = ({coordinates, setShouldTrack}: VehicleIconProps) => {
         setShouldTrack(true);
       }}
     >
-      <MapboxGL.SymbolLayer
+      <MapboxGL.CircleLayer
         id="icon"
```

Comment:
I'm not quite sure, but I believe these id's should be unique. If a different shape source adds a layer with this id 'icon' I think it may cause undesirable effects.